res/values/strings.xml

```diff
@@ -1439,68 +1426,68 @@
     <string name="preferences_storage__photos">Photos</string>
     <string name="preferences_storage__videos">Videos</string>
     <string name="preferences_storage__files">Files</string>
     <string name="preferences_storage__audio">Audio</string>
-    <string name="preferences_storage__review_storage">Review storage</string>
-    <string name="preferences_advanced__use_system_emoji">Use system emoji</string>
+    <string name="preferences_storage__review_storage">View all stored files</string>
+    <string name="preferences_advanced__use_system_emoji">Use Android emoji</string>
     <string name="preferences_advanced__disable_signal_built_in_emoji_support">Disable Signal\'s built-in emoji support</string>
-    <string name="preferences_advanced__relay_all_calls_through_the_signal_server_to_avoid_revealing_your_ip_address">Relay all calls through the Signal server to avoid revealing your IP address to your contact. Enabling will reduce call quality.</string>
+    <string name="preferences_advanced__relay_all_calls_through_the_signal_server_to_avoid_revealing_your_ip_address">For unlisted contacts calls are always routed through the Signal server to avoid revealing your IP address to your contact. Enabling this will also relay calls with listed contacts, but it will reduce call quality.</string>
     <string name="preferences_advanced__always_relay_calls">Always relay calls</string>
     <string name="preferences_app_protection__app_access">App access</string>
     <string name="preferences_app_protection__communication">Communication</string>
     <string name="preferences_chats__chats">Chats</string>
     <string name="preferences_notifications__messages">Messages</string>
     <string name="preferences_notifications__events">Events</string>
-    <string name="preferences_notifications__in_chat_sounds">In-chat sounds</string>
+    <string name="preferences_notifications__in_chat_sounds">Also sound if Signal is open already</string>
     <string name="preferences_notifications__show">Show</string>
     <string name="preferences_notifications__calls">Calls</string>
     <string name="preferences_notifications__ringtone">Ringtone</string>
     <string name="preferences_chats__show_invitation_prompts">Show invitation prompts</string>
-    <string name="preferences_chats__display_invitation_prompts_for_contacts_without_signal">Display invitation prompts for contacts without Signal</string>
+    <string name="preferences_chats__display_invitation_prompts_for_contacts_without_signal">Display a suggestion to invite your contact to Signal, above an SMS chat with a contact who is not on Signal.</string>
     <string name="preferences_chats__message_text_size">Message font size</string>
-    <string name="preferences_events__contact_joined_signal">Contact joined Signal</string>
-    <string name="preferences_notifications__priority">Priority</string>
+    <string name="preferences_events__contact_joined_signal">Contact now uses Signal</string>
```

Comment:
The notification has a timestamp on it so the fact that it is a past event is already clear. I wanted to get rid of the word "joined" because Signal isn't a club, joining it is a weird figure of speech. I don't think it is an issue whether they strictly use it, you could say the same thing for joining. Strictly your contact has registered his phone number.